internal/pginfer/explain.go

```diff
@@ -20,82 +19,39 @@ const (
 
 // Plan is the plan output from an EXPLAIN query.
 type Plan struct {
```

Comment:
I decided to take advantage of the built in JSON parsing to be less stringly typed etc.